src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -49,25 +49,31 @@
  *
  * @param {!Function} childCtor Child class.
  * @param {!Function} parentCtor Parent class.
+ * @param {boolean} useSetPrototypeOf
  */
-$jscomp.inherits = function(childCtor, parentCtor) {
-  /** @constructor */
-  function tempCtor() {}
-  tempCtor.prototype = parentCtor.prototype;
-  childCtor.superClass_ = parentCtor.prototype;
-  childCtor.prototype = new tempCtor();
-  /** @override */
-    childCtor.prototype.constructor = childCtor;
+$jscomp.inherits = function(childCtor, parentCtor, useSetPrototypeOf) {
+  if (useSetPrototypeOf === true && Object.setPrototypeOf) {
+    childCtor.prototype = Object.create(parentCtor.prototype);
+    /** @override */ childCtor.prototype.constructor = childCtor;
+    Object.setPrototypeOf(childCtor, parentCtor);
+  } else {
+    /** @constructor */
+    function tempCtor() {}
```

Comment:
This declaration generates an error in strict mode compilation.

```
[synthetic:es6/util/inherits] :61: ERROR - functions can only be declared at top level or immediately within another function in strict mode
    function tempCtor() {}
    ^^^^^^^^^^^^^^^^^^^^^^
```

I will fix it by changing the line to
```javascript
var tempCtor = function() {};
```